src/util/file_util.cpp

```diff
@@ -24,25 +24,34 @@ Date: January 2012
     defined(__unix__) || \
     defined(__CYGWIN__) || \
     defined(__MACH__)
-#include <sys/stat.h>
+#include <fstream>
+#include <cstring>
+#include <cstdlib>
+#include <algorithm>
+#include <sstream>
+#include <vector>
+
 #include <unistd.h>
 #include <dirent.h>
-#include <cstdlib>
 #include <cstdio>
+#include <sys/stat.h>
```

Comment:
Moving this include looks like a rather random change?

Reply:
I'm not a fan of creating dedicated refactor PRs just to change one line. If such small piggybacked adjustments are discouraged, the code quality is bound to deteriorate over time.

Reply:
I do agree, but I can't seem to make sense of this change? Why is this include better placed down here than where it used to sit?